Common/Securities/SecurityTransactionManager.cs

```diff
@@ -257,6 +257,16 @@ public List<Order> GetOpenOrders(Symbol symbol)
             return _orderProcessor.GetOrders(x => x.Symbol == symbol && x.Status.IsOpen()).ToList();
         }
 
+        /// <summary>
+        /// Get a list of open orders satisfying the filter.
+        /// </summary>
+        /// <returns>List of open orders.</returns>
+        public List<Order> GetOpenOrders(Func<Order, bool> filter)
+        {
+            filter = filter ?? (x => true);
+            return _orderProcessor.GetOrders(x => x.Status.IsOpen() && filter(x)).ToList();
```

Comment:
I'm weary of bloating the API. This behavior is easily accomplished via `Transactions.GetOrders`, although I suppose this is the more common use case.. algos are almost always requesting open orders.